diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index e465d8e..bbd0038 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -197,6 +197,7 @@ module.exports = function (webpackEnv) {
   };
 
   return {
+    ignoreWarnings: [/Failed to parse source map/],
     target: ['browserslist'],
     // Webpack noise constrained to errors and warnings
     stats: 'errors-warnings',
@@ -584,6 +585,14 @@ module.exports = function (webpackEnv) {
                 'sass-loader'
               ),
             },
+            // Shaders
+            {
+              test: /\.(glsl|vs|fs|vert|frag)$/,
+              type: 'asset/source',
+              generator: {
+                filename: 'assets/images/[hash][ext]',
+              },
+            },
             // "file" loader makes sure those assets get served by WebpackDevServer.
             // When you `import` an asset, you get its (virtual) filename.
             // In production, they would get copied to the `build` folder.
@@ -713,7 +722,7 @@ module.exports = function (webpackEnv) {
           // Bump up the default maximum size (2mb) that's precached,
           // to make lazy-loading failure scenarios less likely.
           // See https://github.com/cra-template/pwa/issues/13#issuecomment-722667270
-          maximumFileSizeToCacheInBytes: 5 * 1024 * 1024,
+          maximumFileSizeToCacheInBytes: 10 * 1024 * 1024,
         }),
       // TypeScript type checking
       useTypeScript &&
